src/tv2-common/inewsConversion/converters/ParseBody.ts

```diff
@@ -566,13 +568,18 @@ export function getTransitionProperties(typeStr: string): Pick<PartdefinitionTyp
 	if (transitionMatch) {
 		definition.transition = {
 			style: AtemTransitionStyleFromString(transitionMatch[1].toUpperCase()),
-			duration: transitionMatch[2] ? Number(transitionMatch[2]) : undefined
+			duration: transitionMatch[2] ? getTimeForTransition(transitionMatch[2]) : undefined
 		}
 	}
 
 	return definition
 }
 
+function getTimeForTransition(timeString: string): number {
+	const time = Number(timeString)
+	return time > MAX_ALLOWED_TRANSITION_FRAMES ? MAX_ALLOWED_TRANSITION_FRAMES : time
```

Comment:
The condition can be reduced to a `Math.min` call:
```suggestion
	return Math.min(time, MAX_ALLOWED_TRANSITION_FRAMES)
```